sis_import_build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh --
```

Comment:
very soon after the production deploy of sections project we are going to move the sections tool out of canvas.